lib/src/component.dart

```diff
@@ -62,8 +62,10 @@ abstract class FluxComponent<ActionsT, StoresT> extends react.Component {
     Map<Store, Function> handlers = new Map.fromIterable(redrawOn(),
         value: (_) => (_) => redraw())..addAll(getStoreHandlers());
     handlers.forEach((store, handler) {
-      StreamSubscription subscription = store.listen(handler);
-      _subscriptions.add(subscription);
+      if (store != null) {
+        StreamSubscription subscription = store.listen(handler);
+        _subscriptions.add(subscription);
+      }
```

Comment:
if they're passing in a store that hasn't been initialized yet and we swallow it like this, that will result in an unexpected behavior.  even if the store were eventually initialized, it still wouldn't be wired to the component redraw behavior.

Is there a real use case for this null store bit?